serverless/lambda/src/main/java/uk/gov/di/lambdas/SendNotificationHandler.java

```diff
@@ -114,7 +99,53 @@ public APIGatewayProxyResponseEvent handleRequest(
         }
     }
 
+    private APIGatewayProxyResponseEvent handleVerifyPhoneNumberRequest(
+            SendNotificationRequest sendNotificationRequest, Session session)
+            throws JsonProcessingException {
+        if (sendNotificationRequest.getPhoneNumber() == null) {
```

Comment:
do we want to validate the phone number here https://github.com/google/libphonenumber - otherwise it is going to just go onto the queue and fail x number of times till it dead letters?

Reply:
That is handled in a separate ticket so was planning on leaving that until LTG-409.